input/input.go

```diff
@@ -62,7 +63,9 @@ func (in DefaultHandler) ProcessMetricPoint(point schema.MetricPoint, format msg
 	} else {
 		in.receivedMPNO.Inc()
 	}
-	if !point.Valid() {
+	// in cassandra we store timestamps as 32bit signed integers.
+	// math.MaxInt32 = Jan 19 03:14:07 UTC 2038
+	if !point.Valid() || point.Time >= math.MaxInt32 {
```

Comment:
is this the same for bigtable?

Reply:
in bigtable we use varint encoding and have 8bytes to work with.
I don't have a strong opinion on where this should go. the alternative is we add 2 interface methods to the index plugins, which for bigtable just return true, and in cassandra's case, the validation logic gets split into two different locations. is that an advantegous design? i don't know.

the benefit of using the lowest common denominator is also that it allows data to be moved from one store to another

Reply:
We can always do that later if it becomes an issue, but I think this is fine.